plugin/src/main/kotlin/com/github/triplet/gradle/play/tasks/PublishBundle.kt

```diff
@@ -0,0 +1,74 @@
+package com.github.triplet.gradle.play.tasks
+
+import com.github.triplet.gradle.play.internal.MIME_TYPE_STREAM
+import com.github.triplet.gradle.play.internal.PlayPublishPackageBase
+import com.github.triplet.gradle.play.internal.playPath
+import com.github.triplet.gradle.play.internal.trackUploadProgress
+import com.google.api.client.googleapis.json.GoogleJsonResponseException
+import com.google.api.client.http.FileContent
+import com.google.api.services.androidpublisher.AndroidPublisher
+import com.google.api.services.androidpublisher.model.Bundle
+import org.gradle.api.tasks.InputFile
+import org.gradle.api.tasks.OutputDirectory
+import org.gradle.api.tasks.PathSensitive
+import org.gradle.api.tasks.PathSensitivity
+import org.gradle.api.tasks.TaskAction
+import org.gradle.api.tasks.incremental.IncrementalTaskInputs
+import java.io.File
+
+open class PublishBundle : PlayPublishPackageBase() {
+    @Suppress("MemberVisibilityCanBePrivate", "unused") // Used by Gradle
+    @get:PathSensitive(PathSensitivity.RELATIVE)
+    @get:InputFile
+    val bundle by lazy {
+        // TODO: If we take a customizable folder, we can fix #233, #227
+        File(project.buildDir, "outputs/bundle/${variant.name}/bundle.aab")
```

Comment:
I just built this locally and ran it and my .aab happened to be app.aab. Using AGP 3.2.0-beta01. Probably should not hardcode this one?

Reply:
Yeah, I'm running into the same issue with `beta02` in some production apps. Not in my test project though, still investigating.